jwst/assign_wcs/nircam.py

```diff
@@ -204,20 +250,26 @@ def wfss(input_model, reference_files):
 
     Parameters
     ----------
-    input_model: jwst.datamodels.ImagingModel
+    input_model: `~jwst.datamodels.ImagingModel`
         The input datamodel, derived from datamodels
     reference_files: dict
         Dictionary {reftype: reference file name}.
 
+    Returns
+    -------
+    pipeline : list
+        The pipeline list that is returned is suitable for
+        input into  gwcs.wcs.WCS to create a GWCS object.
+        
```

Comment:
In the Notes section just below, is the first line a complete sentence?  It looks as if the first line should end with a period and the second line should begin with "Not".